Tasks/KubernetesManifestV0/src/models/TaskInputParameters.ts

```diff
@@ -26,4 +28,21 @@ if (!namespace) {
 if (!namespace) {
     tl.debug('Namespace was not supplied nor present in the endpoint; using "default" namespace instead.');
     namespace = 'default';
-}
\ No newline at end of file
+}
+
+export function validateCanaryPercentage() {
+    if (deploymentStrategy.toUpperCase() === canaryDeploymentHelper.CANARY_DEPLOYMENT_STRATEGY && !validateRegex("^(([0-9]|[1-9][0-9]|100)(\\.\\d{1,2})?)$", canaryPercentage)) {
```

Comment:
Should be > 0% and it's not necessary that decimal places be provided only upto 2 places